herddb-thirdparty/openjpa/src/main/java/herddb/openjpa/DBDictionary.java

```diff
@@ -28,9 +28,8 @@
     public DBDictionary() {
         platform = "HerdDB";
         databaseProductName = "HerdDB";
-        supportsForeignKeys = false;
-        supportsUniqueConstraints = false;
-        supportsCascadeDeleteAction = false;
+        supportsDeferredConstraints = false;
```

Comment:
this is the same change did on OpenJPA 3.1.3-SNAPSHOT on "official" HerdDB dictionary